Dockerfile.Windows

```diff
@@ -0,0 +1,9 @@
+ARG BASE_IMAGE
+ARG IMAGE_TAG
+FROM mcr.microsoft.com/windows/${BASE_IMAGE}:${IMAGE_TAG}
+ENV CSI_PROXY_PIPE "\\\\.\\pipe\\csipipe"
```

Comment:
Why do we define the ENV here instead of in the Daemonset spec? Doesn't the pipe path change depending on which API endpint we're talking to?

Reply:
right, it is not needed here. removed it